logos/logos-agent/app/triggers.py

```
@@ -491,6 +495,38 @@ async def _mark_comments_scanned(self, moment: datetime) -> None:
+        file one, and a session handed the title alone can only say that it
+        was handed the title alone.
+
+        Kept: the people who may write to the repository, and the person who
+        opened the issue — who is usually the one with the details, and
+        whose report is why anybody assigned this. Everybody else is a
+        stranger commenting on a public issue, and this session will push a
+        branch.
+        """
+        try:
+            entries, missing = await github.issue_conversation(number)
+        except Exception as exc:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)